crates/provider/src/provider/root.rs

```diff
@@ -36,6 +37,12 @@ impl<T: fmt::Debug, N> fmt::Debug for RootProvider<T, N> {
     }
 }
 
+/// Helper function to directly access [`ProviderBuilder`] with minimal
+/// generics.
+pub fn builder<N: Network>() -> ProviderBuilder<Identity, Identity, N> {
```

Comment:
I think this should be ethereum network like in `ProviderBuilder::new`?

Reply:
That would create cyclic deps

Reply:
How? It's right there in the file

Reply:
Oh wait, I read that as it should be "in ethereum network", as in the network crate.

Reply:
> I think this should be ethereum network like in `ProviderBuilder::new`?

Cannot set defaults in a free function.